### PR TITLE
Computes range separated exchange using improvements suggested by Andy Simmonett

### PR DESCRIPTION
**Pull request by obrien951**
Currently works inside rhf only.

## Description
Improves wall clock time of range separated DFT calculations by storing range separated and regular HF integrals together.

## Todos
Notable points (developer or user-interest) that this PR has or will accomplish.
- [x] Improves integral calculation and storage strategies inside `DFHelper`
- [x] Makes changes to the  `driver` , `libscf_solver`, and the `jk` base class to set the necessary parameters.
- [ ] Intelligent use of tensors to calculate J to save memory
- [ ] Create a class inside libmints to return the desired integrals
- [ ] Extend this functionality to DiskDFJK

## Questions
- [ ] Question1

## Checklist
- [ ] Tests added for any new features
- [ ] [All or relevant fraction of full tests run](http://psicode.org/psi4manual/master/build_planning.html#how-to-run-a-subset-of-tests)

## Status
- [ ] Ready for review
- [ ] Ready for merge

### CODE REVIEW

File: psi4/src/psi4/libfock/MemDFJK.cc [hunk 1]
```diff
@@ -92,7 +94,8 @@ void MemDFJK::compute_JK() {
     if (lr_symmetric_) {
         if (do_wK_) {
             for (size_t N = 0; N < wK_ao_.size(); N++) {
-                wK_ao_[N]->hermitivitize();
+                if (do_wK_) { wK_ao_[N]->hermitivitize();}
+                if (do_K_) { K_ao_[N]->hermitivitize();}
```
Comment (susilehtola) [inline psi4/src/psi4/libfock/MemDFJK.cc]: Note that there's already an if clause at the beginning!

**Comment by hokru** [comment]
oh, looks like the rebase went wrong? Your first commit is still deep in the history.

**Comment by loriab** [comment]
> oh, looks like the rebase went wrong? Your first commit is still deep in the history.

Yeah, Joe and I are in talks.

**Comment by JonathonMisiewicz** [comment]
Is this PR superseded by #1911? If so, don't forget to close this one.

**Comment by obrien951** [comment]
Messed up the Rebase, work moved to PR 1911

**Pull request closed without merging**